libafl/src/inputs/bytes.rs

```diff
@@ -102,6 +102,18 @@ impl From<&[u8]> for BytesInput {
     }
 }
 
+impl From<BytesInput> for Vec<u8> {
+    fn from(value: BytesInput) -> Vec<u8> {
+        value.bytes
+    }
+}
+
+impl From<&BytesInput> for Vec<u8> {
```

Comment:
This one I'm not so sure about: People may end up calling `.into()` in the harness instead of `.target_bytes()` which is wrong for some inputs, and an unnecessary `clone`.
You can use `bytes().to_vec()` for already, which might be more obvious about both facts, what do you think?

Reply:
Oh I see, the `OwnedSlice` wrapper. Make sense, I agree it's better to remove this.

Reply:
Done